pkg/utils/clusterinfo/common.go

```diff
@@ -0,0 +1,20 @@
+package clusterinfo
+
+// ServerVersionInfo is the server version and git_hash.
+type ServerVersionInfo struct {
+	Version string `json:"version"`
+	GitHash string `json:"git_hash"`
+}
+
+type Common struct {
+	DeployCommon
```

Comment:
what does serialized JSON look like if you use this style?

Reply:
I have listed above, you can take a look at: https://github.com/pingcap-incubator/tidb-dashboard/pull/14#issuecomment-590159354